src/test/java/com/fundacionjala/movies/MovieRegularTest.java

```diff
@@ -0,0 +1,51 @@
+package com.fundacionjala.movies;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Charito on 6/13/2016.
+ */
+public class MovieRegularTest {
+    private Movie regularMovie;
+
+    @Test
+    public void movieRegularTitleMustBeTheSpecified() {
+        regularMovie = new MovieRegular("The Revenant");
```

Comment:
Please add a before method for this line. because you are repeating the same instance in all the unit tests
```java
@Before
public void setUp() {
    regularMovie = new MovieRegular("The Revenant");
}
```